### PR TITLE
Feature/logger jobname

### PR DESCRIPTION
**Pull request by shubhanan-ox**

### CODE REVIEW

File: docs/codelab/codelab.json [hunk 1]
```diff
@@ -3,13 +3,13 @@
...
-  "updated": "2019-06-19T00:53:16Z",
+  "updated": "2019-06-27T20:52:35Z",
   "id": "openx-bq-scheduling",
   "duration": 90,
   "title": "Analytics Codelab: Scheduling Queries with your Team Git Repo",
-  "authors": "Jacob Ferriero , Subramanya Gunturu, Brandon O'Connor",
+  "authors": "Jacob Ferriero, Subramanya Gunturu, Brandon O'Connor",
   "summary": "In this codelab, you'll use a python wrapper to define a BigQuery pipeline and schedule it to run on a GKE cluster by checking it into a git repo",
-  "source": "1OOciYhVo9TPcc-amBgzkLxHfQSUK-mnf3x8sCIntyTE",
+  "source": "https://docs.google.com/document/d/1OOciYhVo9TPcc-amBgzkLxHfQSUK-mnf3x8sCIntyTE",
```
Comment (jaketf) [inline docs/codelab/codelab.json]: I think this does not need the whole URL just the id after `d/`

File: tests/test_bqpipeline.py [hunk 2]
```diff
@@ -54,8 +63,24 @@ def test_create_job_config_destination(self):
...
         self.assertEqual(cfg.priority, bigquery.QueryPriority.INTERACTIVE)
-        self.assertEqual(cfg.create_disposition, bigquery.job.CreateDisposition.CREATE_NEVER)
-        self.assertEqual(cfg.write_disposition, bigquery.job.WriteDisposition.WRITE_EMPTY)
+        self.assertEqual(cfg.create_disposition,
+                         bigquery.job.CreateDisposition.CREATE_NEVER)
+        self.assertEqual(cfg.write_disposition,
+                         bigquery.job.WriteDisposition.WRITE_EMPTY)
+
+
+class TestLogging(unittest.TestCase):
```
Comment (jaketf) [inline tests/test_bqpipeline.py]: Thanks for improving test coverage here!

File: README.md [hunk 3]
```diff
@@ -74,10 +74,21 @@ source venv/bin/activate
...
 
+```bash
+pipenv install --python 3
 ```
-python3 -m pip install -r requirements.txt
+
+### Install with pipenv
+
+```bash
+pipenv run python -m unittest discover
```
Comment (jaketf) [inline README.md]: This looks duplicate of below. 
Seems like this should be 
`python3 -m pip install -r requirements.txt` ?
or `pip3 install -r requirements.txt`

File: tests/test_bqpipeline.py (hunk 2 above)
Comment (shubhanan-ox) [reply tests/test_bqpipeline.py]: This was Brandon's doing. Thanks Brandon!

File: docs/codelab/codelab.json (hunk 1 above)
Comment (shubhanan-ox) [reply docs/codelab/codelab.json]: Done. Note that this was updated by klaat.

File: README.md (hunk 3 above)
Comment (shubhanan-ox) [reply README.md]: Done.

**Pull request merged**